app/services/test_application_creation_service.rb

```diff
@@ -31,9 +24,14 @@ def providers
   def create_test_application(provider, trait)
     FactoryBot.create(
       :application,
-      :with_applicant,
+      :with_substantive_scope_limitation,
       trait,
-      provider: provider
+      provider: provider,
+      applicant: passported_applicant
     )
   end
+
+  def passported_applicant
+    FactoryBot.create :applicant, date_of_birth: '10-01-1980', last_name: 'Walker', national_insurance_number: 'JA293483A'
+  end
```

Comment:
Similar to above, I'm not sure why this change is necessary here (sorry if I'm missing something obvious 😁)